projects/core/src/lib/mask.ts

```diff
@@ -275,37 +303,4 @@ export class Maskito extends MaskHistory {
             this.handleInsert(event, '\n');
         }
     }
-
-    private updateSelectionRange([from, to]: SelectionRange): void {
```

Comment:
I have reordered methods so that all `handleWhatever` are at the bottom just for visual grouping reasons.